src/mainwindow.cpp

```diff
@@ -2385,6 +2386,43 @@ void MainWindow::setupActions()
     addAction(action);
     Actions.add("timelineReload", action, tr("Timeline"));
 
+    action = new QAction(tr("Rerun Filter Analysis"), this);
+    connect(action, &QAction::triggered, this, [&](bool checked) {
+        FindAnalysisFilterParser parser;
+        parser.skipAnalyzed(false);
+        // Look for two pass filters
+        if (m_timelineDock->model()->tractor() && m_timelineDock->model()->tractor()->is_valid()) {
+            parser.start(*m_timelineDock->model()->tractor());
+        }
+        if (m_playlistDock->model()->playlist() && m_playlistDock->model()->playlist()->is_valid()) {
+            parser.start(*m_playlistDock->model()->playlist());
+        }
+        if (parser.filters().size() > 0) {
+            QMessageBox dialog(QMessageBox::Question,
+                               windowTitle(),
+                               tr("This will start %1 analysis jobs. Continue?")
```

Comment:
This needs to use the plural form for `tr`
`tr("This will start %1 analysis jobs. Continue?", nullptr, parser.filters().size())`

Reply:
Making another reply in case you did not see it... I changed "jobs" to "job(s)". As I recall, I need to manually update `translations/shotcut_en.ts` to handle the plural forms for English. And I search for "(s)"  to help find them.

Reply:
Done

Reply:
> I changed "jobs" to "job(s)"

I'm glad you mentioned that. Done.